cmd/tailscale/main.go

```diff
@@ -498,6 +498,18 @@ func (a *App) runBackend() error {
 						return nil // even on error. see big TODO above.
 					})
 				})
+				if !hadPrevProtect {
```

Comment:
We should always DebugRebind, the condition here is slightly fiddly and the "had" does not provide us direct insight into / relationship with when the backend last bound its sockets.

Reply:
Agreed and updated